selftests/functional/test_loader.py

```diff
@@ -215,6 +215,9 @@ def test_load_not_a_test(self):
     def test_load_not_a_test_not_exec(self):
         self._test('notatest.py', NOT_A_TEST, 'NOT_A_TEST')
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
And I keep answering that there is an artificial 5s timeout set by us based on our systems and I keep proposing increasing of this timeout. We already had many commits to increase timeouts because our systems were too quick when developing the tests, this is yet another example like that.